package/MDAnalysis/core/groups.py

```diff
@@ -672,34 +672,29 @@ def translate(self, t):
         atomgroup.universe.trajectory.ts.positions[atomgroup.indices] += vector
         return self
 
-    def rotate(self, R, point=None):
+    def rotate(self, R, point=(0, 0, 0)):
         r"""Apply a rotation matrix `R` to the selection's coordinates.
+        :math:`\mathsf{R}` is a 3x3 orthogonal matrix that transforms a vector
+        :math:`\mathbf{x} \rightarrow \mathbf{x}'`:
+
+        .. math::
+
+            \mathbf{x}' = \mathsf{R}\mathbf{x}
 
         Parameters
         ----------
         R : array_like
             3x3 rotation matrix to use for applying rotation.
         point : array_like, optional
-            Center of rotation. If ``None`` then the center of geometry of this
-            group is used.
+            Center of rotation.
 
         Returns
         -------
         self : AtomGroup
 
         Notes
         -----
-        By default (``point=None``) the rotation is performed around
-        the centroid of the group (:meth:`center_of_geometry`). In
-        order to perform a rotation around, say, the origin, use
-        ``point=[0, 0, 0]``.
-
-        :math:`\mathsf{R}` is a 3x3 orthogonal matrix that transforms a vector
-        :math:`\mathbf{x} \rightarrow \mathbf{x}'`:
-
-        .. math::
-
-            \mathbf{x}' = \mathsf{R}\mathbf{x}
+        By default rotates around center of origin ``point=(0, 0, 0)``.
```

Comment:
I added in the notes now how once can chose a different rotation center. For my original case it is already good that the paramter list gives me the option and tells me the default is 0, 0, 0